themes/src/main/resources/theme/base/account/totp.ftl

```diff
@@ -30,7 +30,7 @@
 
 <ol>
     <li>
-        <p>${msg("totpStep1")}</p>
+        <p>${msg("totpStep1")?no_esc}</p>
```

Comment:
There is no links in the message anymore as that has been removed:
https://github.com/keycloak/keycloak/blob/master/themes/src/main/resources/theme/base/account/messages/messages_en.properties#L101

Reply:
You're right. I was irritated because the links are still included in the translations.

Reply:
We're unable to maintain the translations and as such need the community to step up and update as needed. Hence why they still exist in translations.